### DISCUSSION TITLE
When using an Ethernet equipped device/board, default to it instead of Wi-Fi

### DISCUSSION THREAD

**Discussion by maslyankov** [technyon/nuki_hub#524] [Ideas]
Currently, when your device of choice has Ethernet, to configure and use it, you still need to connect to Wi-Fi and so on. This feels totally unnecessary and time consuming. It would be great if a check is added for this to solve this time-spender.

**Comment by iranl** [comment]
Just because a device has ethernet doesn't mean a user is using it.

Besides ethernet also needs to be setup manually to know the right device or to setup custom gpio pin settings.
As this has both technical issues and is a minor one time inconvenience this will not be worked on.

**Comment by technyon** [comment]
@maslyankov I actually agree it would be nice to have a setup without WiFi, but I don't see any easy way to do it. With an ESP there's no Plug&Play on the hardware level like you have on a PC or a USB connection. The ESP just doesn't it has a LAN module attached until the user configures it.
